sources/packages/backend/apps/queue-consumers/src/processors/schedulers/sfas-integration/_tests_/sims-to-sfas-integration.scheduler.e2e-spec.ts

```diff
@@ -814,8 +814,7 @@ describe(describeProcessorRootTest(QueueNames.SIMSToSFASIntegration), () => {
   ): string {
     const offering = application.currentAssessment.offering;
     const offeringIntensity =
-      offering?.offeringIntensity ??
-      application.data.howWillYouBeAttendingTheProgram;
+      offering?.offeringIntensity ?? application.offeringIntensity;
```

Comment:
Minor, the const `const offeringIntensity` can be eliminated and the `application.offeringIntensity` can be used directly instead.